website/src/controllers/api-v3/groups.js

```diff
@@ -563,15 +520,25 @@ api.inviteToGroup = {
     let uuids = req.body.uuids;
     let emails = req.body.emails;
```

Comment:
Hmm probably a good idea. At some point we'll also have to enforce limits in other places too